doc/start/hardware-recommendations.rst

```diff
@@ -54,11 +54,11 @@ Monitors and managers (ceph-mon and ceph-mgr)
 Monitor and manager daemon memory usage generally scales with the size of the
 cluster.  Note that at boot-time and during topology changes and recovery these
 daemons will need more RAM than they do during steady-state operation, so plan
-for peak usage.  For very small clusters, 32 GB suffices.  For
-clusters of up to, say, 300 OSDs go with 64GB.  For clusters built with (or
-which will grow to) even more OSDS you should provision
-128GB.  You may also want to consider tuning settings like ``mon_osd_cache_size``
-or ``rocksdb_cache_size`` after careful research.
+for peak usage. For very small clusters, 32 GB suffices. For clusters of up
+to, say, 300 OSDs go with 64GB. For clusters built with (or which will grow
+to) even more OSDS you should provision 128GB.  You may also want to consider
+tuning settings like ``mon_osd_cache_size`` or ``rocksdb_cache_size`` after
```

Comment:
While we're in the neighborhood, suggest 
tuning settings including :confval:\`mon_osd_cache_size\` or :confval:\`rocksdb_cache_size\` after

Reply:
I'm on the fence about this kind of semantic markup. What do we gain for the extra cost of this semantic correctness?

Reply:
I'm toying with the idea of providing these explicit confvals in all these sections, and I see the advantage: all the information about the configuration is there. I have the neophyte's hesitancy about this whole enterprise, though, because of the huge cost of marking up all the variables in the documentation.

Also: We're going to need a confval definition of rocksdb_cache_size whether we choose to undertake this markup project.

Reply:
This kind of cross-referencing does make the documentation stronger, and I am grateful for the suggestion.